src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -267,6 +267,16 @@ public void elementShouldBeDisabled(String locator) {
         }
     }
 
+    @RobotKeyword("Verify the element identified by ``locator`` is focused\r\n" +
+            "\r\n" +
+            "Key attributes for arbitrary elements are id and name. See `Introduction` for details about log levels and locators.")
+    @ArgumentNames({ "locator" })
+    public void elementShouldBeFocused(String locator) {
+        if (!isFocused(locator)) {
+            throw new SeleniumLibraryNonFatalException(String.format("Element %s is disabled.", locator));
```

Comment:
Should the message says "isn't focused" instead of "is disabled"?

Reply:
copy paste error...
Doint too many things at once, as usual...

Reply:
and fixed..